### ISSUE TITLE
Cannot set the Configuration for an application in Kubernetes Multi-app run

### ISSUE THREAD

**Issue by msfussell** [dapr/cli#1351]
## Expected Behavior
Cannot get Zipkin to show traces from the mult-app run on K8s deployment. The deployment is missing the configuration annotations. This dapr.io/config | Tells Dapr which Configuration resource to use
From here https://docs.dapr.io/reference/arguments-annotations-overview/

## Actual Behavior
The multi-app run K8s template https://v1-12.docs.dapr.io/developing-applications/local-development/multi-app-dapr-run/multi-app-template/ does not have this property that can be set in the template. 
These are the annotations that you see for the deployed applications. Note there is not config annotation.
```
Name:         pythonapp-54bdcfbd48-rg94r
Namespace:    default
Priority:     0
Node:         mark-control-plane/172.19.0.2
Start Time:   Mon, 02 Oct 2023 20:47:29 -0700
Labels:       app=pythonapp
              dapr.io/app-id=pythonapp
              dapr.io/metrics-enabled=true
              dapr.io/sidecar-injected=true
              pod-template-hash=54bdcfbd48
Annotations:  dapr.io/app-id: pythonapp
              dapr.io/enabled: true
              dapr.io/metrics-port: 65307
              dapr.io/unix-domain-socket-path:

Name:         nodeapp-56998d7c99-fhz8q
Namespace:    default
Priority:     0
Node:         mark-control-plane/172.19.0.2
Start Time:   Mon, 02 Oct 2023 20:47:25 -0700
Labels:       app=nodeapp
              dapr.io/app-id=nodeapp
              dapr.io/metrics-enabled=true
              dapr.io/sidecar-injected=true
              pod-template-hash=56998d7c99
Annotations:  dapr.io/app-id: nodeapp
              dapr.io/app-port: 3000
              dapr.io/enabled: true
              dapr.io/metrics-port: 65295
              dapr.io/unix-domain-socket-path:
```

## Steps to Reproduce the Problem

- Run the K8s Hello world tutorial with K8s Multi-app run PR so that traces are being written
- Do `kubectl port-forward svc/zipkin 19411:9411`
- Go to http://localhost:19411/ and there are no traces written

## Release Note
RELEASE NOTE: **UPDATE** Cannot set the Configuration for an application in Kubernetes Multi-app run

**Comment by mukundansundar** [comment]
Currently there is no way to set the config annotation, only the default config can be used. 
The default config, does not seem to be set, we can add a fix for setting the default config.

**Comment by mukundansundar** [comment]
If the `appconfig` Dapr configuration is not found in `default` namespace for some reason, the `dapr.io/config` parameter will not be set and skipped emitting a warning: 
```
⚠  Dapr configuration "appconfig" not found in namespace "default". Skipping annotation "dapr.io/config"
```

**Comment by mukundansundar** [comment]
<img width="1543" alt="image" src="https://github.com/dapr/cli/assets/65565396/ce607d0b-4932-47c0-90ee-0778bafdb967">
I have made the changes locally and tested ....

**Comment by mukundansundar** [comment]
closed by #1352
